cmd/hostscheduler/scheduler/types.go

```diff
@@ -0,0 +1,23 @@
+package scheduler
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+const (
+	ShootLabel = "testmachinery.sapcloud.io/host"
+)
+
+func ShootKubeconfigSecretName(shootName string) string {
```

Comment:
Although file is named types.go, here are no types. Is this the right place for this functions?